Dockerfile.import

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:bionic
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    ca-certificates curl gnupg && rm -rf /var/lib/apt/lists/*
+
+RUN echo 'deb http://ppa.launchpad.net/osmadmins/ppa/ubuntu bionic main\n\
+deb-src http://ppa.launchpad.net/osmadmins/ppa/ubuntu bionic main' > \
+    /etc/apt/sources.list.d/osmadmins-ppa.list
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
+    --recv A438A16C88C6BE41CB1616B8D57F48750AC4F2CB
```

Comment:
Use `--recv-key` instead of `--recv`. Otherwise it will fail.

Reply:
I could not get it to work with the bionic image, even with `--recv-key`. I updated to Debian 12 Bookworm instead.